packages/framework/aqueduct/src/data-object-factories/pureDataObjectFactory.ts

```diff
@@ -171,7 +183,10 @@ export class PureDataObjectFactory<
 	 *
 	 * @param context - data store context used to load a data store runtime
 	 */
-	public async instantiateDataStore(context: IFluidDataStoreContext, existing: boolean) {
+	public async instantiateDataStore(
+		context: IFluidDataStoreContext,
+		existing: boolean,
+	): Promise<FluidDataStoreRuntime> {
```

Comment:
This leaves me wondering if it should return the interface, but I see in the current type definition file that it's currently typed like this, so 🤷.

Reply:
Yeah. I'm trying to avoid actual API changes in these PRs as much as possible. But that definitely seems like a good change. I'll do that in a subsequent PR.

Reply:
#19353